packages/docs/src/routes/docs/faq/index.mdx

```diff
@@ -209,3 +210,13 @@ An amazing team of contributors around the world living in [Discord](https://qwi
 ## Is Qwik open source?
 
 Yes, [MIT](https://github.com/BuilderIO/qwik/blob/main/LICENSE) and [dependency-free](https://github.com/BuilderIO/qwik/blob/main/packages/qwik/package.json), installing Qwik will not bloat your node_modules nor your lawyers.
+
+# Does Qwik have any downside?
+
+Yes. Every framework, possessing its own strengths and weaknesses, involves a trade off.
+
+1. As a relative newcomer to the JS world, Qwik has yet to develop a community of libraries and best practices.
+2. Qwik’s magic lies in its serialization, but this also brings about some limitations developers wouldn’t otherwise have to consider.
```

Comment:
```suggestion
2. Qwik’s magic lies in its serialization, but this also brings about some considrations developers wouldn’t otherwise have to be mindful of.
```

Feels best to call this "considerations" vs "limitations". Esp large companies will be wary of "limits" in qwik, which imo in a worst case scenario Qwik operates just like the current status quo of frameworks. It's not a "limit" to the framework imo, aka you should never find a case where you use Qwik and say "oh it has limits that are a problem for our business so we need to migrate off", but instead things to consider (esp around serialization), esp when you want the best possible perf benefit

Reply:
```suggestion
2. Qwik’s magic lies in its serialization, but this also brings about some considerations developers wouldn’t otherwise have to be mindful of.
```

Reply:
That's true. And I correct "considrations" to "considerations" :)